packages/core/src/scatterplot.ts

```diff
@@ -38,69 +44,83 @@ function generateMessages (
       anchor: getAnchor(spec.type, visElement),
       requires: ['type'],
       text: `The chart Is based on colored ${spec.type?.value} elements.`,
-      title: 'Interacting with the chart',
-      onboardingStage: interacting,
+      title: 'Reading the chart',
+      onboardingStage: reading,
       marker: {
         id: 'unique-marker-id-2'
       },
       id: 'unique-message-id-2',
-      order: 1
+      order: 2
     },
+
     {
       anchor: getAnchor(spec.legendTitle, visElement),
       requires: ['legendTitle'],
-      text: `The legend shows the ${spec.legendTitle?.value} for the chart. The colors range from blue to white and brown.`,
+      text: `The legend shows the <i>${spec.legendTitle?.value} </i> for the chart. The colors range from blue to white and brown.`,
       title: 'Reading the chart',
       onboardingStage: reading,
       marker: {
         id: 'unique-marker-id-3'
       },
       id: 'unique-message-id-3',
-      order: 1
+      order: 4
     },
     {
-      anchor: getAnchor(spec.xAxisTitle, visElement),
-      requires: ['xAxisTitle', 'yAxisTitle'],
-      text: `The columns show the ${spec.xAxis?.value}, while the rows show the ${spec.yAxis?.value}.`,
+      anchor: getAnchor(spec.yAxisTitle, visElement),
+      requires: ['yAxisTitle', 'xAxisTitle'],
+      text: `The <i>${spec.yAxisTitle?.value} (y-axis)</i> for a certain <i>${spec.xAxisTitle?.value} </i>.`,
       title: 'Reading the chart',
-      onboardingStage: analyzing,
+      onboardingStage: reading,
       marker: {
-        id: 'unique-marker-id-4'
+        id: 'unique-marker-id-5'
       },
-      id: 'unique-message-id-4',
-      order: 2
+      id: 'unique-message-id-5',
+      order: 3
     },
     {
-      anchor: getAnchor(spec.yAxisTitle, visElement),
-      requires: ['yAxisTitle', 'xAxisTitle'],
-      text: `the ${spec.yAxisTitle?.value} (y-axis) for a certain ${spec.xAxisTitle?.value}.`,
-      title: 'Interacting with the chart',
+      anchor: getAnchor(spec.interactDesc, visElement),
+      requires: ['interactDesc'],
+      text: 'Hover over the chart to get the dedicated value for each data point.',
+      title: 'Interaction with the chart',
       onboardingStage: interacting,
       marker: {
-        id: 'unique-marker-id-5'
+        id: 'unique-marker-id-6'
       },
-      id: 'unique-message-id-5',
-      order: 2
+      id: 'unique-message-id-6',
+      order: 1
     },
     {
       anchor: getAnchor(spec.maxValue, visElement),
-      requires: ['maxValue'],
-      text: `The chart Is based on colored ${spec.type?.value} elements.`,
-      title: 'Reading the chart',
-      onboardingStage: reading,
+      requires: ['maxValue', 'maxX', 'maxY'],
+      text: `The data point (${spec.maxX?.value}, ${spec.maxY?.value}) is the farest point from the origin.`,
```

Comment:
Can we change the message as 'This dot (230,16) is closest to the origin'. (for min)                                                                               This is the dot() with largest distance from the origin(for max)

Reply:
Actually I would like to prevent "origin" here as the origin does not have any relevance in this most scattercharts (It's only in HIDDEN for the scatterchart representing the data in a latent space).

Therefore, I'd suggest somethin like: "The highest horsepower of 230 relates to 16 miles per gallon."